Themes/default/Errors.template.php

```diff
@@ -225,7 +225,7 @@ function template_show_file()
 		$is_target = $line_num == $context['file_data']['target'];
 		echo '
 			<tr>
-				<td align="right"', $is_target ? ' class="current">==&gt;' : '>', $line_num , ':</td>
+				<td ', $is_target ? ' class="current righttext" : "righttext">==&gt;' : '>', $line_num , ':</td>
```

Comment:
This is incorrect. This will cause it to output `<td class="current righttext" : "righttext">==&gt;"` if $is_target is true. If the order in which the classes are defined doesn't matter then you can just do this: `<td class="righttext', $is_target ? ' current">==&gt;' : '">'`. Otherwise you'll need to do it like this: `<td ', $is_target ? 'class="current righttext">==&gt;' : 'class="righttext">'`